components/html/modules/content-patterns/_typography.html

```diff
@@ -0,0 +1,50 @@
+@@include('../navigation/_breadcrumbs.html')
+<main id="main" class="content">
+	<div class="container">
+	    <div class="row">
+	    	<div class="maincontent col-lg-9 col-md-9 col-sm-9 col-xs-12 right">
+				<h1>H1 Champion Welterweight 38px #98700d</h1>
+				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Explicabo, impedit repudiandae maxime. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
+				<h2>H2 Archivo Narrow 32px #98700d</h2>
+				<p>Explicabo, impedit repudiandae maxime.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
+				<h3>H3 Archivo Narrow Bold 28px #707070</h3>
+				<p>Obcaecati aliquid, aspernatur exercitationem. Explicabo, impedit repudiandae maxime. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla. Lorem ipsum dolor sit amet, consectetur adipisicing elit.  Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime.</p>
+				<h4>H4 Archivo Bold 24px Caps #98700d</h4>
+				<p>Dolor sit amet, consectetur adipisicing elit. Obcaecati aliquid, aspernatur exercitationem. Explicabo, impedit repudiandae maxime. Commodi, quod earum voluptatem dolorem aliquam omnis repellendus vel quisquam similique dolores, esse maxime. <a href="#">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a> Debitis impedit suscipit cum ea cumque reiciendis laudantium sed, aut unde optio molestiae non aliquam officia aliquid est earum minus qui nulla.</p>
+				<aside class="highlight">
```

Comment:
I wasn't sure what class name to give this style - the gray text highlight

Reply:
I think that name works just fine.